tests/core/process.c

```diff
@@ -117,9 +125,31 @@ static gboolean malloc_range_check_cb (
     const GumMallocRangeDetails * details, gpointer user_data);
 #endif
 
+GThread * new_sleeping_dummy_thread_sync (volatile gboolean * done)
```

Comment:
Return type should be on a separate line unless it's a prototype.

Reply:
Also, this helper function is lower level than the functions useing it, so it should be below them.